static/js/apps/eval_retrieval_generation/sxs/app.tsx

```diff
@@ -24,23 +24,38 @@ import { getDocInfo } from "../util";
 import { SessionContext } from "./context";
 import { getLeftAndRight } from "./left_right_picker";
 import { QueryWithTables } from "./query_with_tables";
+import { SxsFeedback } from "./sxs_feedback";
 
 interface AppPropType {
   sessionId: string;
   sheetIdA: string;
   sheetIdB: string;
 }
 
+function getSortedQueryIds(docInfos: { a: DocInfo; b: DocInfo }) {
+  const idsA = Object.keys(docInfos?.a?.allQuery || []);
```

Comment:
should be `|| {}` since allQuery is an object?

Reply:
ah true, changed!